pixl_imaging/README.md

```diff
@@ -24,3 +24,21 @@ pip install -e ../pixl_core/ .
 ## Usage
 
 Usage should be from the CLI driver, which interacts with the endpoint.
+
+
+## Configuration and database interaction
+
+The database tables are updated using alembic, see the [alembic](alembic) dir for more details.
+
+The `SKIP_ALEMBIC` environmental variable is used to control whether migrations are applied to the database.
+
+- Tests that don't use the database use `SKIP_ALEMBIC=true`, but otherwise you probably want to run this.
+- If you wanted to test out new migrations from a test/dev deployment on the GAE you may want to set
+skip alembic migrations
```

Comment:
I don't get this line, it reads to me as "if you want to test migrations, skip them"

Reply:
That's fair, how does this look?
```suggestion
- If you want to ensure that a data migration setp works from the `pixl_test` or `pixl_dev` deployment on the GAE you have two choices:
1. pull in the new version with migrations and only redeploy the `pixl-imaging` container (keeping postgres up), or
2. Manually run each step yourself by skipping migrations being run on startup of the container 
  - pull in the code with the new migration(s)
```

Reply:
I think I'm missing the reasoning for wanting to test the migration step and how that is different from normal deployment

Reply:
I guess if you have some changes to columns which require manpulating the data, you may want to test that if you had data in that table, that the migration is applied successfully. If that makes sense?

Reply:
sort of, but why wouldn't this work with normal deployment? Do we wipe the database every time we deploy?

Reply:
it would. I guess the point is that only production would I expect to always be up, test and dev you'd probably wipe because you don't care about them. So this gives you a path to test this before rolling out to the production database. 

May be trying to prematurely optimise anyway, generally I expect we're not going to be doing migrations which require altering data. I'd expect we just add or remove columns and tables